apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -123,68 +123,94 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
             'grid-cols-3': id === 'entity-details',
           })}
         >
-          {formData?.map(({ title, isEditable, type, format, pattern, value, dropdownOptions }) =>
```

Comment:
Wrapped the return value by curly brackets so there are lots of difference, but I'll mark what specifically I changed in this file.